### PR TITLE
Add basic Ul

### PR DESCRIPTION
**Pull request by Baroka-wp**
In this PR we implement the following :
- [ ] Create a `app.rb` file that serve as console app entry-point. 

- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.

- [ ] List all rentals for a given person `id`.
- [ ] In `main.rb` define the entry point
- [ ] Present the user with a list of options to perform.
- [ ] Lets users choose an option.
- [ ] If needed, ask for parameters for the option.
- [ ] Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
 # capitalizedPerson = CapitalizeDecorator.new(person)
 # p capitalizedPerson.correct_name
 # capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
 # p capitalizedTrimmedPerson.correct_name
+
+b = Book.new('way', 'Yun')
+student = Student.new(nil, 12, 'Rio')
+student.add_rental('2021', b)
+student.add_rental('2022', b)
+p student.rentals
```
Comment (ahmetbozaci) [inline main.rb]: image

- [ ] Kindly dnt commit unused codes. [Requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @Baroka-wp  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- App is working as expected. I can add people, books, and rentals and ı can display them.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: main.rb (hunk 1 above)
Comment (Baroka-wp) [reply main.rb]: ## thank you @ahmetbozaci

**Review by Munsa1 — APPROVED** [review]
Hi @Baroka-wp, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Baroka-wp** [comment]
## Thank you @Munsa1

**Pull request merged**